plugins/kubernetes-backend/src/cluster-locator/LocalKubectlProxyLocator.ts

```diff
@@ -26,7 +26,7 @@ export class LocalKubectlProxyClusterLocator
     this.clusterDetails = [
       {
         name: 'local',
-        url: 'http:/localhost:8001',
+        url: 'http://localhost:8001',
```

Comment:
I'm not entirely sure I follow what is triggering the actual issue here. Is it that we don't know whether the kube proxy is listening to IPv4 or IPv6, and there are situations where only one of them will be available?

I wonder if we could do the resolution upfront here in that case? either if we could do a lookup for example with https://nodejs.org/api/dns.html#dnslookuphostname-options-callback, or if we just try to connect over both 4 and 6 and pick one that succeeds?

Reply:
This is doing nothing to fix the issue, just fixing what is very likely to have been a typo.

Reply:
Yeah I wasn't talking about this exact change, but rather the issue you're aiming to fix. I'm suggesting we implement the logic I proposed in this part of the code